mkdocs.yml

```
@@ -29,6 +29,7 @@ nav:
       - 'Use Cases': 'getting-started/use-cases.md'
       - 'Running Renovate': 'getting-started/running.md'
       - 'Installing & Onboarding': 'getting-started/installing-onboarding.md'
+      - 'Troubleshooting Renovate': 'getting-started/troubleshooting.md'
```

```suggestion
      - 'Troubleshooting': 'getting-started/troubleshooting.md'
```

We could also get away with just `Troubleshooting`? Not sure how verbose the sidebar items should be... 😉

---

I think shorter is better